README.md

```diff
@@ -1 +1,45 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## ABOUT ME
+
+![Kagwiria Nkonge.](https://github.com/Nkonge/hw01/blob/master/Images/Profile_pic.jpg)
+
+<center>
+
+My name is Nkonge Kagwiria Regina, a graduate student (class of 2020) at Harris School of Public Policy, *studying MA International Development and Policy. I come from Kenya and I am an Obama Foundation Scholar. My policy interests are international development, security, conflict and humanitarian intervention as well as technology and education*. 
+Prior to coming to Harris my work focused on technology for development particularly the application of technology in higher education, refugee camps and hard to reach areas. I hold an MSc. in Information Systems Security and Digital Forensics.
+**I am excited to be in this class!**
+
+</center>
+
+## PURPOSE OF THIS PROJECT
+<center>
+
+The goal of this project is to test your software installation, our GitHub setup, and our homework submission process via pull request, as well as demonstrate competency in Markdown.
+
+</center>
+
+## HOW I CREATED THIS FILE
+
+    1. I installed Git (R studio was already installed in my computer)
+    2. I forked and cloned the hw01 repository
+    3. I opened the README file using R Studio and Git as the version control software
+    4. I edited the README using RMarkdown: added the following sections - about me, purpose of this project, how I created this file
+    5. I staged and commited the file, then pushed it to GitHub
+ 
+## REFLECTION
+<center>
+
+The process was a bit confusing because at first I was using GitHub desktop, only to realize that Git and GitHub are different. 
+Secondly, I had at first, created a separate file from the README, and practised RMarkdown skills on the separate file. I later realized that I can edit the README directly using RMarkdown and that there was no need to create a different file. 
+
+</center>
+
+## **LINKS THAT I FOUND USEFUL**
+
+[hw01 Video](https://cfss.uchicago.edu/homework/edit-readme/)
+
+[Getting Started With R Markdown](https://rstudio.com/resources/webinars/getting-started-with-r-markdown/)
+
+[R Markdown Cheat Sheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) 
+
```

Comment:
Good reflection! In future assignments you can just leave this as a comment in the pull request itself, rather than any of the homework documents